common/db.go

```diff
@@ -213,9 +260,6 @@ func (db *DB) Close() {
 	if db.selectOrchs != nil {
 		db.selectOrchs.Close()
 	}
-	if db.filterOrchs != nil {
```

Comment:
How can I extract that into it's seperate commit since it's already part of this commit? thanks !

Reply:
You can use `git add -p` which triggers git's patch mode (see this [blog post](https://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/) for a more in-depth treatment of the command).

Suppose 8f4ac86 is the latest HEAD commit and the PR is approved already. You can do the following:

```
# This will unstage all the changes for latest HEAD commit
git reset HEAD~1

# Run git add in patch mode
git add -p
```

If the commit that you care about is further back then you can either use `HEAD~N` where N is the # of commits back from the HEAD or you can reference the commit hash.

The last command will open an interactive console that allows you to select "hunks" (basically chunks of code changes that Git detects) to stage for a commit. You can stage the hunk that corresponds to this change and then add a specific commit for it.